microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -223,30 +217,30 @@ To make your application use a Microsoft Edge preview channel by using an enviro
 
-    Alternatively, set the **Variable name** to `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE`, and set the **Variable value** to `1`.
+    Alternatively, set the **Variable name** to `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE`, and set the **Variable value** to `1`, as explained below.
 
 1.  Select **OK** to close the dialog boxes.
 
-    :::image type="complex" source="./media/env-vars-new-user-variable.png" alt-text="Adding a new environment variable, as a user variable" lightbox="./media/env-vars-new-user-variable.png":::
-       Adding a new environment variable, as a user variable
-    :::image-end:::
+    :::image type="content" source="./media/env-vars-new-user-variable.png" alt-text="Adding a new environment variable, as a user variable" lightbox="./media/env-vars-new-user-variable.png":::
 
-> [!NOTE]
-> This approach sets the environment variable for all WebView2 apps, not just the app you're testing.  To set this environment variable for only the WebView2 app which you are testing, if you're running your app from the command prompt, set the environment variable `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1`.  That sets the environment variable just for the current `cmd.exe` command-prompt process and for any new child processes from that `cmd.exe` instance.  Then the environment variable only applies to the WebView2 app that you're testing.
+This approach sets the environment variable for all WebView2 apps, not just the app you're testing.  To set this environment variable for only the WebView2 app which you are testing, if you're running your app from the command prompt, set the environment variable `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE=1`.  That sets the environment variable just for the current `cmd.exe` command-prompt process and for any new child processes from that `cmd.exe` instance.  Then the environment variable only applies to the WebView2 app that you're testing.
 
-> [!NOTE]
-> After setting an environment variable this way, the environment variable is applied to any new processes that are created.  The environment variable doesn't apply to processes which are already running.  To ensure that all processes use the new environment variable, you may need to restart Visual Studio or log out of Windows and then log in again.
+After setting an environment variable this way, the environment variable is applied to any new processes that are created.  The environment variable doesn't apply to processes which are already running.  To ensure that all processes use the new environment variable, you may need to restart Visual Studio or log out of Windows and then log in again.
+
+
+### Values for WEBVIEW2_RELEASE_CHANNEL_PREFERENCE
+
+| Value | Description |
+|---|---|
+| `1` | Use a [preview channel](https://www.microsoftedgeinsider.com/download) of Microsoft Edge (Beta, Dev, or Canary). |
+| Value other than `1` | Use the stable release of Microsoft Edge. |
```

Comment:
This should refer to the search order rather than specific channels. For example, 1 (reverse order) if only the stable runtime is installed, will still choose the stable. Similarly, the default search order could result in using Canary if stable is not installed.